packages/cadl-ranch-specs/http/type/property/optionality/main.tsp

```diff
@@ -112,19 +112,19 @@ model DurationProperty is ModelTemplate<duration>;
 @operationGroup
 interface Duration extends OperationsTemplate<DurationProperty, "\"P123DT22H14M12.011S\""> {}
 
-// Model with optional plaindate property
-@doc("Model with a plaindate property")
-model PlaindateProperty is ModelTemplate<plainDate>;
-@route("/plaindate")
+// Model with optional plainDate property
+@doc("Model with a plainDate property")
+model PlainDateProperty is ModelTemplate<plainDate>;
+@route("/plainDate")
 @operationGroup
-interface Plaindate extends OperationsTemplate<PlaindateProperty, "\"2022-12-12\""> {}
+interface PlainDate extends OperationsTemplate<PlainDateProperty, "\"2022-12-12\""> {}
```

Comment:
Em, that actually change the client name...

Reply:
yeah and this is the purpose because plaindate is not actually a word.
I wonder why the cspell here does not report it when we introduce it